src/app/mappers/concerts-upcoming-response-to-concert-upcoming-items/concerts-upcoming-response-to-concert-upcoming-items.mapper.ts

```diff
@@ -0,0 +1,20 @@
+import { ConcertUpcomingItem } from '../../interfaces/concert-upcoming-item.interface';
+import { ConcertsUpcomingResponse } from '../../interfaces/concerts-upcoming-response.interface';
+
+export const mapConcertsUpcomingResponseToConcertUpcomingItems = (
+  concertUpcomingResponse: readonly ConcertsUpcomingResponse[]
+): readonly ConcertUpcomingItem[] =>
+  concertUpcomingResponse.map((concertUpcomingItemTo: ConcertsUpcomingResponse) => ({
+    imageDesktopSource: concertUpcomingItemTo.ImageDesktopSource,

```

Comment:
📓 itt ha destruktalod az objektumot, rovidebb lehet a kod, itt ott, nem muszaj megcsinalni csak tipp, bar elegge elteroek a nevek... na mindegy, ez a te dontesded, hagyhatod igy is

Reply:
Ezt most kihagyom, de köszönöm az észrevételt